frontend/src/v5/store/currentUser/currentUser.selectors.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { createSelector } from 'reselect';
+
+const selectCurrentUserDomain = (state) => (state.currentUser2);
+
+export const selectCurrentTeamspace = createSelector(
```

Comment:
Lets get rid of this one because we are not using it and it could be confused with another currentTeamspace